config/webpack.common.js

```diff
@@ -103,7 +103,8 @@ module.exports = function ({ env, metadata }) {
           'LAUNCHER_KEYCLOAK_URL' : JSON.stringify(metadata.LAUNCHER_KEYCLOAK_URL),
           'LAUNCHER_KEYCLOAK_REALM' : JSON.stringify(metadata.LAUNCHER_KEYCLOAK_REALM),
           'LAUNCHER_KEYCLOAK_CLIENT_ID': JSON.stringify(metadata.LAUNCHER_KEYCLOAK_CLIENT_ID),
-          'LAUNCHER_FRONTEND_SENTRY_DSN': JSON.stringify(metadata.LAUNCHER_FRONTEND_SENTRY_DSN)
+          'LAUNCHER_FRONTEND_SENTRY_DSN': JSON.stringify(metadata.LAUNCHER_FRONTEND_SENTRY_DSN),
+          'LAUNCHER_STANDALONE': JSON.stringify(metadata.LAUNCHER_STANDALONE)
```

Comment:
So `LAUNCHER_STANDALONE=true` will make the Zip download option disappear? 
That looks a bit confusing, given that there are cases where ZIP is still needed even in standalone mode. Perhaps rename it to be more explicit?

Reply:
It will remove the entire target-environment step from the flow:
![screenshot from 2018-10-24 14-41-21](https://user-images.githubusercontent.com/51133/47430927-05ae1e80-d79b-11e8-8bb6-4342c2e4541d.png)
I have no clue what this should be called do you have a suggestion?